libs/langgraph/langgraph/pregel/__init__.py

```diff
@@ -609,6 +610,36 @@ def config_specs(self) -> list[ConfigurableFieldSpec]:
             ]
         ]
 
+    def config_schema(
+        self, *, include: Optional[Sequence[str]] = None
+    ) -> Type[BaseModel]:
+        # If the config type is not set explicitly OR
+        # if the checkpointer is turned on, then we'll want to infer the config.
+        # When the checkpointer is turned on, additional fields are injected
+        # into the configurable object which correspond to the checkpointing parameters.
+        if self.config_type is None or self.checkpointer is not None:
```

Comment:
My understanding is that when the config schema is provided, we want to ignore any auto behavior w/ the exception of the checkpointer fields? (i.e., we do want to expose thread_id as part of the configurable schema?)